Update totalconnect.markdown

austinmroczek opened the pull request:
Add info about the arm_away_instant and arm_stay_instant services

## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->



## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/60156
- Link to parent pull request in the Brands repository: n/a
- This PR fixes or closes issue: n/a

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

austinmroczek committed:
Update totalconnect.markdown

Add info about the arm_away_instant and arm_stay_instant services provided by https://github.com/home-assistant/core/pull/60156

frenck commented on source/_integrations/totalconnect.markdown:
```diff
@@ -66,7 +66,7 @@ sudo apt install libxml2-dev libxmlsec1-dev
 The integration provides an Alarm Control Panel for each TotalConnect location. It uses the name of your location from TotalConnect.  For example, if your location name in TotalConnect is "Home", then you will get `alarm_control_panel.home` in Home Assistant.
 
-The alarm control panel supports the following services: `alarm_arm_away`, `alarm_arm_home`, `alarm_arm_night` and `alarm_disarm`.
+The alarm control panel supports the following services: `alarm_arm_away`, `alarm_arm_home`, `alarm_arm_night`, and `alarm_disarm`.  The integration also provides unique services for `arm_stay_instant` and `arm_away_instant` which arm the system with zero entry delay, triggering the alarm instantly if someone opens a door.
```
```suggestion
The alarm control panel supports the following services: `alarm_arm_away`, `alarm_arm_home`, `alarm_arm_night`, and `alarm_disarm`. The integration also provides unique services for `arm_stay_instant` and `arm_away_instant` which arm the system with zero entry delay, triggering the alarm instantly if someone opens a door.
```

frenck replied:
```suggestion
The alarm control panel supports the following services: `alarm_arm_away`, `alarm_arm_home`, `alarm_arm_night`, and `alarm_disarm`. The integration also provides unique services for `arm_stay_instant` and `arm_away_instant` which arm the system with zero entry delay, triggering the alarm instantly if someone opens a door.
```

austinmroczek committed:
Update source/_integrations/totalconnect.markdown

Co-authored-by: Franck Nijhof <git@frenck.dev>

austinmroczek committed:
Update totalconnect.markdown

Match arm_home_instant change in code (instead of arm_stay_instant).

austinmroczek committed:
Update totalconnect.markdown

Add an example service call, and highlight that the unique services are called via totalconnect.* instead of alarm_control_panel.*

austinmroczek commented:
I updated to "arm_home_instant" to match changes in the code.  I also added an example automation using the new service, and tried to highlight that it is called differently (totalconnect.* instead of alarm_control_panel.*)
